File: backend/src/main/java/com/ssafy/ganhoho/domain/auth/dto/LoginResponse.java
package com.ssafy.ganhoho.domain.auth.dto;

import lombok.*;

@Getter
@Builder
@AllArgsConstructor(access = AccessLevel.PROTECTED)
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class LoginResponse {
    private Long memberId;
    private String loginId;
    private String name;
    private String hospital;
    private String ward;
    private Double hospitalLng;
    private Double hospitalLat;
    private String accessToken;
    private String refreshToken;
}

File: ganhoho_app/app/src/main/java/com/ssafy/ganhoho/data/model/response/auth/LoginResponse.kt
package com.ssafy.ganhoho.data.model.response.auth

data class LoginResponse(
    val memberId: Long,
    val loginId: String,
    val name: String,
    val hospital: String?,
    val ward: String?,
    val accessToken: String,
    val refreshToken: String,
    val hospitalLat: Double?,
    val hospitalLng: Double?
)